### PR TITLE
GO-2041 | JS-2979 Sync widget update on account creation

### PR DESCRIPTION
**Pull request by KirillSto**
---

- [x] I understand that contributing to this repository will require me to agree with the [CLA](https://github.com/anyproto/.github/blob/main/docs/CLA.md)

---

### Description

This PR fixes built-in usecases addition flow by sending events on widget blocks creation synchronously.

### What type of PR is this? (check all applicable)

- [ ] 🍕 Feature
- [x] 🐛 Bug Fix
- [ ] 📝 Documentation Update
- [ ] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build
- [ ] 🔁 CI

### Related Tickets & Documents

https://linear.app/anytype/issue/JS-2979/the-first-block-with-space-info-in-the-sidebar-is-absent

### Added tests?

- [ ] 👍 yes
- [x] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

### Added to documentation?

- [ ] 📜 README.md
- [ ] 📓 [tech-docs](https://github.com/anyproto/tech-docs)
- [x] 🙅 no documentation needed

### CODE REVIEW

**Review by fat-fellow — APPROVED** [review]

File: core/block/editor/widget/widget.go [hunk 1]
```diff
@@ -106,3 +107,12 @@ func (w *widget) CreateBlock(s *state.State, req *pb.RpcBlockCreateWidgetRequest
 
 	return wrapper.Model().Id, nil
 }
+
+func (w *widget) CreateBlocks(s *state.State, reqs []*pb.RpcBlockCreateWidgetRequest) error {
```
Comment (deff7) [inline core/block/editor/widget/widget.go:111]: don't over-complicate api

Comment (KirillSto) [reply core/block/editor/widget/widget.go:111]: @deff7 I was solving the problem with single Apply on request. Previous version used to invoke CreateWidgetBlock from 4 to 6 times, that triggers state.Apply every time and rewites the list of events. The main aim here is to send events on creating Widget blocks synchronously, so either we should check messages list in context inside Apply, or create separate command that will create multiple blocks and invoke Apply one single time

Comment (fat-fellow) [reply core/block/editor/widget/widget.go:111]: We need some kind of bulking ops tho

Comment (deff7) [reply core/block/editor/widget/widget.go:111]: CreateBlock doesn't apply changes, DoStateCtx does

**Review by deff7 — APPROVED** [review]

**Pull request merged**